tests/end-to-end/grafana/datasources.cy.js

```diff
@@ -109,10 +109,19 @@ describe("grafana dev datasources", function() {
   })
 
   it("has workload cluster", function() {
-    cy.contains("Workload Cluster")
-      .should("exist")
-      .siblings()
-      .contains("default")
-      .should("exist")
+    cy.yqDigParse("sc", ".global.clustersMonitoring")
+      .then(([first_cluster, ...rest_clusters]) => {
+        cy.contains(`Workload Cluster ${first_cluster}`)
```

Comment:
Is this test working now? Doesn't this also need a check if it only is one WC or not?

Reply:
True, thought I tested but think i had the wrong naming still then. Fixed now, for some reason I also had to revert my change that added `parent()`. I tested this with both single and multi wc.